### ISSUE TITLE
[BITV] 9.1.1.1a/4.1 - The "Close" button of the image editor does not have an alternative text, which could be provided, for example, using the "aria-label"-attribute. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41820]
![image](https://github.com/nextcloud/server/assets/6078378/e7cd7712-ac4e-4432-b28e-fbecbb163f68)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-cd239865a5-v4-n1

**Comment by ShGKme** [comment]
The issue is located in the viewer which uses external library `filerobot-image-editor` for the editor.

**Comment by JuliaKirschenheuter** [comment]
closing as not relevant anymore
